webpack/assets/javascripts/react_app/components/LoginPage/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { LoginPage as PFLoginPage } from 'patternfly-react';
+import { adjustProps } from './LoginPageHelpers';
+import './LoginPage.scss';
+
+const LoginPage = props => <PFLoginPage {...adjustProps(props)} />;
+
+LoginPage.propTypes = {
+  alert: PropTypes.objectOf(PropTypes.string),
+  backgroundUrl: PropTypes.string,
+  caption: PropTypes.string,
+  logoSrc: PropTypes.string,
+  token: PropTypes.oneOfType([PropTypes.string, PropTypes.number]).isRequired,
+  version: PropTypes.string,
+};
+
+LoginPage.defaultProps = {
+  alert: null,
+  backgroundUrl: '/assets/login_background.jpg',
```

Comment:
@ShimShtein @ekohl 
Do you think putting the `backgroundUrl` and `logoSrc` in `proptypes`
would be easier to change for branding?

Reply:
discussing and testing it offline with @ShimShtein seems that it works just fine :)